build.sbt

```diff
@@ -90,7 +90,7 @@ lazy val `bsp-testkit` = project
     bashScriptExtraDefines += """addJava "-Dscript.path=${app_home}/"""" + executableScriptName.value,
     batScriptExtraDefines += """call :add_java "-Dscript.path=%APP_HOME%\\"""" + executableScriptName.value + ".bat",
     libraryDependencies ++= List(
-      "org.scalacheck" %% "scalacheck" % "1.14.0",
+      "org.scalacheck" %% "scalacheck" % "1.15.2",
```

Comment:
The new version of scalacheck allows us to use the method `suchThat` that allows allows for a better control how the shrinking actually happens